hw/ip/aes/README.md

```diff
@@ -1,6 +1,16 @@
 # AES HWIP Technical Specification
 
-{{#block-dashboard aes}}
+[`aes/masked`](https://reports.opentitan.org/hw/ip/aes_masked/dv/latest/report.html):
+![](https://dashboards.lowrisc.org/badges/dv/aes/masked/test.svg)
+![](https://dashboards.lowrisc.org/badges/dv/aes/masked/passing.svg)
+![](https://dashboards.lowrisc.org/badges/dv/aes/masked/functional.svg)
+![](https://dashboards.lowrisc.org/badges/dv/aes/masked/code.svg)
+
+[`aes/unmasked`](https://reports.opentitan.org/hw/ip/aes_unmasked/dv/latest/report.html):
```

Comment:
Wouldn't this suggestion make it one row though?

Reply:
The trailing backslash adds a linebreak

Reply:
TIL in markdown this is a linkbreak instead of line continuation.

In this case how about:
```suggestion
![](https://dashboards.lowrisc.org/badges/dv/aes/masked/code.svg)
\
[`aes/unmasked`](https://reports.opentitan.org/hw/ip/aes_unmasked/dv/latest/report.html):
```
? Still have the visual separation but is a `<br/>` instead of a new `<p>`?

Reply:
Why do you prefer to avoid trailing backslashes? I don't believe there are any other (permitted) ways to add a linebreak without a <p> otherwise?

If we like the seperation, then just keep as-is IMO.